src/test/java/helper/Base.java

```diff
@@ -0,0 +1,17 @@
+package helper;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Base {
+
+        public static WebDriver driver=new ChromeDriver();
+        public static void closeSession() {
+                driver.quit();
+        }
+        public static void init(String url){
```

Comment:
Maybe should be better to have first the init session then the close session?

Reply:
done